elixir-europe-training/Training-Metrics-Database
# Feat/add event specific metrics upload

This PR tries to improve the user experience related to uploading event metrics.

It will:
- Add a view which allows a user to upload metrics assuming all data is connected to the specified event
- Add links to the event specific upload view in both the event list and in the "Edit event" view
- Add link to list of uploaded events when events have been successfully uploaded
- Improve UX related to tab indications

![Screen Shot 2024-10-08 at 18 58 06](https://github.com/user-attachments/assets/73506555-94e6-4fbb-97f1-6e5eda89f318)


To test:
- Upload some events
- Use the event list link to view a list of your uploaded events
- Click "Upload metrics" for one of the events
- Upload metrics
- Go back to the event and make sure the metrics have been uploaded
